### PR TITLE
Switch doc links to version 4.1

### PR DESCRIPTION
**Pull request by spadgett**
/assign @zherman0 
/cc @benjaminapetersen

### CODE REVIEW

File: pkg/console/subresource/configmap/brand_ocp.go [hunk 1]
```diff
@@ -4,5 +4,5 @@ package configmap
 
 const (
 	DEFAULT_BRAND   = "ocp"
-	DEFAULT_DOC_URL = "https://docs.openshift.com/container-platform/4.0/"
+	DEFAULT_DOC_URL = "https://docs.openshift.com/container-platform/4.1"
```
Comment (zherman0) [inline pkg/console/subresource/configmap/brand_ocp.go]: Trailing slash is required for all doc links
```
documentationBaseURL:
                  pattern: ^$|^((https):\/\/?)[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/?))\/$
                  type: string
                  description: Documentation base url can optionally be set but must end in a trailing slash
```

**Comment by zherman0** [comment]
@spadgett - All the doc links need trailing slashes

**Comment by spadgett** [comment]
D'oh, that's right. Thanks, updated.

**Pull request merged**